xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -149,6 +148,12 @@ class GemmAutotuner {
 
   absl::StatusOr<AutotuneResult> TuneGpuBlasLt(const HloInstruction* gemm,
                                                const GemmConfig& gemm_config) {
+    TF_ASSIGN_OR_RETURN(
+        auto workspace_shape,
+        ShapeUtil::TryGetSubshape(gemm->shape(),
+                                  {gemm->shape().tuple_shapes_size() - 1}));
+    TF_ASSIGN_OR_RETURN(auto workspace_buffer, CreateBuffer(*workspace_shape));
```

Comment:
This seems to be creating an error: use of undeclared identifier 'CreateBuffer'; did you mean 'DataBuffer'?
  150 |     TF_ASSIGN_OR_RETURN(auto workspace_buffer, CreateBuffer(*workspace_shape));

Reply:
Oh, the error is due to not rebasing to the latest code, should resolve now.